### DISCUSSION TITLE
MatrixCamp 2022 discussion

### DISCUSSION THREAD

**Discussion by dengn** [matrixorigin/matrixone#2132] [Q&A]
Dear all, MatrixOne has launched a developer event for implementing built-in functions and aggregate functions. All developers are welcome to participate. Successful merges will be rewarded. You can checkout this event briefing: https://github.com/matrixorigin/matrixone/issues/1997

You can ask your questions in this discussion regarding this event. The community will respond to you.

**Comment by noneback** [comment]
Is there any good first issue about the storage engine rather than built-in function in the following camp? I am quite interested in that area recently.
